utils/logger/logger.go

```diff
@@ -0,0 +1,120 @@
+package logger
+
+import (
+	"sync"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/lavanet/lava/utils/allowList"
+	"github.com/rs/zerolog"
+	zerologlog "github.com/rs/zerolog/log"
+)
+
+var (
+	instance *Logger
+	once     sync.Once
+)
+
```

Comment:
lets add a comment describing what these numbers are for

Reply:
Fixed 3dca57a241b5d5c718a4202326fd4c7ea711dae7